boefjes/boefjes/plugins/kat_snyk/check_version.py

```diff
@@ -170,4 +170,4 @@ def check_version_in(version: str, versions: str):
     in_range = False
     while not in_range and all_versions:
         in_range, all_versions = check_version_agains_versionlist(version, all_versions)
-    return in_range
+    return in_range if version else False
```

Comment:
```
if not version:
  return False
```
 can be the first line of this method. Or better yet, we could never call this method at all of there is no version.

Reply:
hmm that makes sense yes, not the smartest solution this...